erisdb/pipe/transactor.go

```diff
@@ -212,92 +212,6 @@ func (this *transactor) TransactAndHold(privKey, address, data []byte, gasLimit,
 	return ret, rErr
 }
 
-func (this *transactor) Send(privKey, toAddress []byte, amount int64) (*Receipt, error) {
```

Comment:
Interesting :)  I'll take that as a veto, and make sure we don't deprecate it unless more arguments are brought to the table :)